nixos/modules/services/web-apps/comfyui.nix

```diff
@@ -0,0 +1,139 @@
+{ config, lib, options, pkgs, ...}:
+
+with lib;
+
+let
+  cfg = config.services.comfyui;
+  defaultUser = "comfyui";
+  defaultGroup = defaultUser;
+  mkComfyUIPackage = cfg: cfg.package.override {
+    modelsPath = "${cfg.dataPath}/models";
+    inputPath = "${cfg.dataPath}/input";
+    outputPath = "${cfg.dataPath}/output";
+    customNodes = cfg.customNodes;
+  };
+in
+{
+  options = {
+    services.comfyui = {
+      enable = mkEnableOption
+        (mdDoc "The most powerful and modular stable diffusion GUI with a graph/nodes interface.");
```

Comment:
`lib.mdDoc` is now just an alias and can be safely removed everywhere.
see https://github.com/NixOS/nixpkgs/commit/d36f950d405369abca5144b14a201bc25c4a080a and https://github.com/NixOS/nixpkgs/pull/237557